### PR TITLE
92 fix functions and notebooks

### LINKED ISSUE #92: Fix function problems and update notebooks
With the changes in the functions' structures, several functions and notebooks broke. Fix and update them

### PR DESCRIPTION
**Pull request by CamSalda**
Closes #92

### CODE REVIEW

File: laos_gggi/data_functions/combine_data.py [hunk 1]
```diff
@@ -21,6 +21,45 @@ def load_all_data():
     # 2. EM-DAT data representing the event damages (index: Year, ISO3)
     merged_dict["emdat_damage"] = emdat["df_inten_filtered_adjusted"]
 
+    emdat["df_inten_filtered_adjusted_hydro"] = emdat[
+        "df_inten_filtered_adjusted_hydro"
+    ].drop(columns=["Region", "Subregion"])
+    for x in emdat["df_inten_filtered_adjusted_hydro"].columns:
+        emdat["df_inten_filtered_adjusted_hydro"].rename(
```
Comment (jessegrabowski) [inline laos_gggi/data_functions/combine_data.py:28]: `columns` can be a function in rename:

```
emdat["df_inten_filtered_adjusted_hydro"] =( emdat
    .loc[:, "df_inten_filtered_adjusted_hydro"]
    .drop(columns=["Region", "Subregion"])
    .rename(columns = lambda x: f'{x}_clim')
)
```

File: laos_gggi/replication_data.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    )
+    disasters["hydrological_disasters"] = (
+        hydro_disasters.fillna(0).astype(int).sum(axis=1)
+    )
+
+    # Fill NaN values for disasters and emdat_damage
+    disasters = disasters.fillna(0)
+    emdat_damage = emdat_damage.fillna(0)
+    emdat_damage_hydro = emdat_damage_hydro.fillna(0)
+    emdat_damage_clim = emdat_damage_clim.fillna(0)
```
Comment (jessegrabowski) [inline laos_gggi/replication_data.py:35]: Didn't we decide this was wrong?

**Review by jessegrabowski — APPROVED** [review]
Small comments

File: notebooks/Paper_replication.ipynb [hunk 3]
```diff
@@ -46,12 +46,12 @@
   },
```
Comment (jessegrabowski) [inline notebooks/Paper_replication.ipynb:46]: <p>If this error is fixed you need to re-run the notebook</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/laos-climate-change/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: laos_gggi/data_functions/combine_data.py (hunk 1 above)
Comment (CamSalda) [reply laos_gggi/data_functions/combine_data.py:28]: Nice! The lambda functions are a super power I need to work on. 

I implemented it with a small change, as emdat is a dictionary and does not have .loc[]

emdat["df_inten_filtered_adjusted_clim"] =  (emdat[
    "df_inten_filtered_adjusted_clim"
].drop(columns=["Region", "Subregion"]).rename(columns = lambda x: f'{x}_clim'))

File: laos_gggi/replication_data.py (hunk 2 above)
Comment (CamSalda) [reply laos_gggi/replication_data.py:35]: Yes, you are completely right. Deleted.

File: notebooks/Paper_replication.ipynb (hunk 3 above)
Comment (CamSalda) [reply notebooks/Paper_replication.ipynb]: Done

**Pull request merged**